modules/nw-route-specific-annotations.adoc

```diff
@@ -16,55 +16,122 @@ To create a whitelist with multiple source IPs or subnets, use a space-delimited
-|`ingresses.config/cluster ingress.operator.openshift.io/hard-stop-after` | You can set either an IngressController or the ingress config . This annotation redeploys the router and configures the HA proxy to emit the haproxy `hard-stop-after` global option, which defines the maximum time allowed to perform a clean soft-stop. | `ROUTER_HARD_STOP_AFTER`
-|`router.openshift.io/haproxy.health.check.interval`| Sets the interval for the back-end health checks. (TimeUnits) | `ROUTER_BACKEND_CHECK_INTERVAL`
+and "-". The default is the hashed internal key name for the route. 
+|
+|
+
+|`haproxy.router.openshift.io/pod-concurrent-connections`
+|Sets the maximum number of connections that are allowed to a backing pod from a router. +
+Note: If there are multiple pods, each can have this many connections.  If you have multiple routers, there is no coordination among them, each may connect this many times. If not set, or set to 0, there is no limit. 
+|
+|
+
+|`haproxy.router.openshift.io/rate-limit-connections`
+|Setting `'true'` or `'TRUE'` enables rate limiting functionality which is implemented through stick-tables on the specific backend per route. +
+Note: Using this annotation provides basic protection against distributed denial-of-service (DDoS) attacks. 
+|
+|
+
+|`haproxy.router.openshift.io/rate-limit-connections.concurrent-tcp`
+|Limits the number of concurrent TCP connections made through the same source IP address. It accepts a numeric value. +
+Note: Using this annotation provides basic protection against distributed denial-of-service (DDoS) attacks.
+|
+| 
+
+|`haproxy.router.openshift.io/rate-limit-connections.rate-http`
+|Limits the rate at which a client with the same source IP address can make HTTP requests. It accepts a numeric value.  +
+Note: Using this annotation provides basic protection against distributed denial-of-service (DDoS) attacks.
+|
+|
+
+|`haproxy.router.openshift.io/rate-limit-connections.rate-tcp`
+|Limits the rate at which a client with the same source IP address can make TCP connections. It accepts a numeric value.  +
+Note: Using this annotation provides basic protection against distributed denial-of-service (DDoS) attacks.
+|
+|
+
+|`haproxy.router.openshift.io/timeout` 
+|Sets a server-side timeout for the route. (TimeUnits) 
+|`spec.tuningOptions.serverTimeout` 
+|
+
+|`haproxy.router.openshift.io/timeout-tunnel` 
+|This timeout applies to a tunnel connection, for example, WebSocket over cleartext, edge, reencrypt, or passthrough routes. With cleartext, edge, or reencrypt route types, this annotation is applied as a timeout tunnel with the existing timeout value. For the passthrough route types, the annotation takes precedence over any existing timeout value set. 
```

Comment:
I think this description could be clearer: 
* I cannot tell whether "for example" applies to "WebSocket" or to "WebSocket over cleartext, edge, reencrypt, or passthrough routes".  
* For passthrough routes, the tunnel timeout always applies (see [the HAProxy documentation for `timeout tunnel`](http://cbonte.github.io/haproxy-dconv/2.6/configuration.html#4-timeout%20tunnel)).
* I don't know what "applied as a timeout tunnel" means.  Maybe that can be deleted unless you have some idea what it means?  
* Change "types" to "type" in "the passthrough route types".  
* Instead of "takes precedence over any existing timeout value set", why not write "takes precedence over `haproxy.router.openshift.io/timeout`"?
```suggestion
|This timeout applies to a tunnel connection, such as WebSocket, over cleartext, edge, or reencrypt routes. For the passthrough route type, the annotation always applies and takes precedence over `haproxy.router.openshift.io/timeout`. 
```

Reply:
Done. Let me change this accordingly.